src/main/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/OverhangFixingManager.java

```diff
@@ -68,15 +84,17 @@
      * @param maxMismatchesInOverhangs max number of mismatches permitted in the overhangs before requiring clipping
      * @param maxBasesInOverhangs      max number of bases permitted in the overhangs before deciding not to clip
      * @param doNotFixOverhangs        if true, don't clip overhangs at all
+     * @param processSecondaryReads    if true, allow secondary reads to
```

Comment:
make it clear what happens to secondary reads if this is false, are they filtered or are they passed through unchanged?